test/Microsoft.ML.TimeSeries.Tests/TimeSeriesDirectApi.cs

```diff
@@ -543,12 +567,241 @@ public void AnomalyDetectionWithSrCnn()
             int k = 0;
             foreach (var prediction in predictionColumn)
             {
+                Assert.Equal(3, prediction.Prediction.Length);
+                if (k == 20)
+                    Assert.Equal(1, prediction.Prediction[0]);
+                else
+                    Assert.Equal(0, prediction.Prediction[0]);
+                k += 1;
+            }
+        }
+
+        [Fact]
+        public void AnomalyDetectionWithSrCnnEntireAnomalyOnly()
+        {
+            var ml = new MLContext(1);
+
+            // Generate sample series data with an anomaly
+            DateTime currentTime = new DateTime(2020, 01, 01);
+            var data = new List<TimeSeriesDataWithTimestamp>();
+            for (int index = 0; index < 20; index++)
```

Comment:
Can we add a test that demonstrates loading data from a file and computing anomalies?

Reply:
> Can we add a test that demonstrates loading data from a file and computing anomalies?

Added.